inspec-core.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'train-core', '~> 2.0'
   spec.add_dependency 'license-acceptance', '>= 0.2.13', '< 2.0'
   spec.add_dependency 'thor', '~> 0.20'
-  spec.add_dependency 'json', '>= 1.8', '< 3.0'
+  spec.add_dependency 'json-schema'
```

Comment:
Unbounded dependencies will bite us. Please bound the top end with with `~> x.y`

Reply:
Ok, I figured I'd get called out on this one but I wasn't sure the right minimum here. Thanks!